Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue3343.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+
+namespace Xamarin.Forms.Controls.Issues
+{
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Github, 3343, "[Android] Cursor position in entry and selection length not working on 3.2.0-pre1", PlatformAffected.Android | PlatformAffected.iOS)]
+	public class Issue3343 : TestContentPage
+	{
+		protected override void Init()
+		{
+			Entry entry = new Entry()
+			{
+				Text = "Initialized with CursorPosition and SelectionLength"
+			};
+
+			Entry entry2 = new Entry()
+			{
+				Text = "Click Button to set CursorPosition and SelectionLength"
+			};
+
+			entry.CursorPosition = 4;
+			entry.SelectionLength = entry.Text.Length;
+
+			Content = new StackLayout()
+			{
+				Padding = 20,
+				Children =
+						{
+						new Label{ Text = "The first Entry should have all text selected starting at character 4. Click the button to trigger the same selection in the second Entry." },
```

Comment:
It might be a good idea to add some text here about UWP.  For UWP I had to hit *TAB* for it to select the first entry and see the highlighting at all

Reply:
Yeah, that's a bug. Fix is on the way!